app/src/main/AndroidManifest.xml

```diff
@@ -17,6 +17,15 @@
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
         </activity>
+<<<<<<< HEAD
```

Comment:
Is this still in the code? There weren't have been any merge issues at that point so I'm confused where this came from.